apps/gql/package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "@kampus-apps/gql",
+  "name": "@kampus/gql",
```

Comment:
Just to confirm then "kampus-apps" anything under "apps" (except ui ? )

Reply:
including ui.

- `packages/ui` -> `@kampus/ui`
- `apps/ui` -> `@kampus-apps/ui` -> this will evolve to a docs website for "kampus ui"

if `apps/ui` makes it weird (i think it does) we can rename it to `apps/studio` and release it under https://studio.dev.kamp.us.

but for now let's keep them as `@kampus-apps/*` for all the folders under `apps`

Reply:
Oh i know it's named `@kampus/ui-next` atm, but i want to rename it to `@kampus/ui` as soon as we get rid of its deprecated version.

Reply:
deprecated version is not used i'll change to deprecated.

I'd modify as following

```
@kampus-apps/studio : reference to ui from "apps"
@kampus-apps/gql
@kampus-apps/kampus
@kampus-apps/pasaport
```

what do you think? @usirin

Reply:
I'll move "apps/ui" to "apps/studio" as well so it's not confusing

Reply:
> deprecated version is not used i'll change to deprecated.
> 
> I'd modify as following
> 
> ```
> @kampus-apps/studio : reference to ui from "apps"
> @kampus-apps/gql
> @kampus-apps/kampus
> @kampus-apps/pasaport
> ```
> 
> what do you think? @usirin

This setup is perfect dude. I am also ok with renaming `packages/ui` to `@kampus/ui` too. Let's be done with it once an for all.